client/src/components/Navlinks/Navlinks.js

```diff
@@ -0,0 +1,73 @@
+import React from "react";
+import { Link } from "react-router-dom";
+import "./style.css";
+
+class LogOutNavlinks extends React.Component {
+  render() {
+    return (
+      <div>
+        <li>
```

Comment:
There's a lot of repetition going on here. that means you can have a component and have the props sent down. to that component. extract it out.